CONTRIBUTING.md

```diff
@@ -0,0 +1,46 @@
+## Development
+
+First of all, thank you for your interest in this project. Please read the documentation here to begin contributing. 
+
+### Creating an Action
+
+- :file_folder: In a new branch create an new folder in a root folder of this repository and name it to your Action name.
+- :octocat: Follow the instructions [Creating an Action with the Toolkit](https://github.com/actions/toolkit#creating-an-action-with-the-toolkit) to create an Action.
+- :construction: Start developing and open a pull request.
+
+Read more about GitHub Actions [here](https://help.github.com/en/articles/getting-started-with-github-actions).
+
+### Recommendations on versioning your action
+
+Since we keep multiple actions in one repository, all the actions are released in one batch (we do not support semantic versioning yet). 
+
+**JavaScript / TypeScript Action**
+
+When your pull request was merged:
+
+- create a new branch
+- comment out `node_modules` in `.gitignore`
+
+```
+# comment out in distribution branches
+# node_modules/
+```
+- add **only** your production dependencies:
+
+```bash
+$ npm prune --production
+$ git add node_modules
+$ git commit -a -m "Add production dependencies"
+$ git push
+```
+
+- open a pull request with a destination branch to `latest` branch
+- everything is ready to be :ship:
+
+**Docker Action**
+
+When your pull request was merged:
```

Comment:
Same here. Does this need to say “To start your pull request:”?

Reply:
> When your pull request was merged:

Sorry this one is not clear :// The idea is that when a pull request with an Action implementation is merged into `master` branch, we are ready to release it. To release this Action, we need to open a pull request to the `latest` branch. @janpio pointed out that maybe it should be out of scope for the pull request author. I agree with that 👍 

So you can think of it as instructions for releasing new Action.  Maybe we should create another doc file like `RELEASE.md` or something and put `Recommendations on versioning your action` section there, thought? 🤔